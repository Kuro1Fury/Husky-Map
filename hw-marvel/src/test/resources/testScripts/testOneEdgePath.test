# Tests that one edge graph works as expected

CreateGraph A

AddNode A n1
AddNode A n2

AddEdge A n1 n2 theOnlyEdge

FindPath A n1 n2

FindPath A n2 n1