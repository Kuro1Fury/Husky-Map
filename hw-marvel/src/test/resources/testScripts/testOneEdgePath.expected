# Tests that one edge graph works as expected

created graph A

added node n1 to A
added node n2 to A

added edge theOnlyEdge from n1 to n2 in A

path from n1 to n2:
n1 to n2 via theOnlyEdge

path from n2 to n1:
no path found